### ISSUE TITLE
In the /Locations view please style the Items differently to the Locations

### ISSUE THREAD

**Issue by rebreatherccr** [hay-kot/homebox#770]
### What is the problem you are trying to solve with this feature?

To be able to distinguish quickly the Items separately from the Locations

### What is the solution you are proposing?

Theme the Locations (and/or Items) with Bold or Font Colour/Size Or Background Colour or Inverse

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [X] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by hay-kot** [comment]
They have pretty different icons, is it necessary to add more styles?

![CleanShot 2024-02-16 at 09 05 55@2x](https://github.com/hay-kot/homebox/assets/64056131/5b3af9b0-29d4-4831-9947-be3028935725)

**Comment by rebreatherccr** [comment]
Yes - I see the icons -  I think visually they would look really nice as 2 styles.  Perhaps only a simple colour a subtle dark blue for the items.

I only run the binary ./homebox at the moment on my Mac minus docker

I will try to build a local runtime and edit the Vue on my Mac and paste some options
